recipe/build.sh

```diff
@@ -2,6 +2,16 @@
 
 export LDFLAGS="-L${PREFIX}/lib"
 export CFLAGS="${CFLAGS} -O3 -I${PREFIX}/include"
+
+if [ "$(uname)" == "Darwin" ];
+then
+    export LD_LIBRARY_PATH=$PREFIX/lib:$LD_LIBRARY_PATH
```

Comment:
Could be wrong, but doubt it would make a difference as `brew` installs to `/usr/local/lib`; so, expect that gets picked up no matter what.